src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/ElasticSearchAnalysisGraph.java

```diff
@@ -88,13 +88,16 @@
 public class ElasticSearchAnalysisGraph extends AnalysisGraph {
 
   private static final Logger LOG = LogManager.getLogger(ElasticSearchAnalysisGraph.class);
+  private static final int rcaPeriod = 12;
+  private static final int evaluationIntervalSeconds = 5;
+
 
   @Override
   public void construct() {
-    Metric heapUsed = new Heap_Used(5);
-    Metric gcEvent = new GC_Collection_Event(5);
-    Metric heapMax = new Heap_Max(5);
-    Metric gc_Collection_Time = new GC_Collection_Time(5);
+    Metric heapUsed = new Heap_Used(evaluationIntervalSeconds);
```

Comment:
Ideally we should remove this argument in constructor and fix the value to 5 seconds in base class. Joydeep wanted us to refactor this long time ago

Reply:
I can see that this the evaluationInterval of the AggregateMetric is 1s. I dont know if there is any reason for that. If not, we should remove it altogether from here and fix its value to constant 5 seconds.

Reply:
let's create a separate issue to keep track of that.